admin/class-pw-admin-admin.php

```
@@ -0,0 +1,2016 @@
+         * class.
+         */
+
+        wp_enqueue_style($this->plugin_name, plugin_dir_url(__FILE__) . 'css/pw-admin-admin.css', array(), $this->version, 'all');
+    }
+
+    /**
+     * Register the JavaScript for the admin area.
+     *
+     * @since    1.0.0
+     */
+    public function enqueue_scripts($hook)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$hook'. (Unused Code Rules)

(UnusedFormalParameter)